src/server.c

```diff
@@ -1074,14 +1075,23 @@ static bool handle(qd_server_t *qd_server, pn_event_t *e, pn_connection_t *pn_co
                     qd_format_string(conn_msg, sizeof(conn_msg), "[C%"PRIu64"] Connection to %s failed: %s %s",
                                      ctx->connection_id, config->host_port, pn_condition_get_name(condition),
                                      pn_condition_get_description(condition));
+
+                    qd_format_string(conn_msg_1, sizeof(conn_msg_1), "Connection to %s failed: %s %s",
+                                     config->host_port, pn_condition_get_name(condition), pn_condition_get_description(condition));
                 } else {
                     qd_format_string(conn_msg, sizeof(conn_msg), "[C%"PRIu64"] Connection to %s failed",
                                      ctx->connection_id, config->host_port);
+                    qd_format_string(conn_msg_1, sizeof(conn_msg_1), "Connection to %s failed", config->host_port);
+                }
```

Comment:
Can you just add a comment here explaining that we're trying to filter out loading the logs with duplicate messages if the connection cannot come up?  Include the github issue for extra points, thanks.

Reply:
Done !! thanks